packages/ckeditor5-image/src/imageinsert/imageinsertui.js

```diff
@@ -113,7 +113,7 @@ export default class ImageInsertUI extends Plugin {
 					insertImageViaUrlForm.label = t( 'Insert image via URL' );
 				}
 			}
-		} );
+		}, { priority: 'low' } );
```

Comment:
Could you add a comment like here:  
https://github.com/ckeditor/ckeditor5/blob/fa810d440464e112bbb5428895c161b09ae9fd26/packages/ckeditor5-media-embed/src/mediaembedui.js#L71-L73